src/server/database/Updater/DBUpdater.cpp

```diff
@@ -178,7 +178,7 @@ bool DBUpdater<T>::Create(DatabaseWorkerPool<T>& pool)
     // Path of temp file
     static Path const temp("create_table.sql");
 
-    // Create temporary query to use external mysql cli
+    // Create temporary query to use external MySQL CLi
```

Comment:
Hmm, shouldnt the **i** be capital as well as it is short for command line interface, no?